### PR TITLE
WIP: Stripping

### PR DESCRIPTION
**Pull request by apuignav**
I commit this barebones lesson because that's the only thing it's missing, taking into account comments in #83. I'll do it today, but ideas are more than welcome.

Also, if somebody has a nice flowchart of the dataflow, plz share.

### CODE REVIEW

**Comment by apuignav** [comment]
Travis fail expected until #59 is merged.

File: 08-stripping.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Collisions recorded by the LHCb detector go through a very specific data flow in order to ensure the maximum data-taking efficiency and data quality.
+In each step, specific [Gaudi](01-davinci.html)-based applications process the data and produce the output for the next step.
+The data flow is designed so the processing of real data and simulated one are as close to each other as possible, with small differences that will be discussed when applicable:
+
+ 1. (Only in simulation) The _pp_ collisions are generated using `Pythia` (currently `Pythia8`), decays of hadronic particles are described by `EvtGen` and the interaction of the generated particles with the detectors and its response are implemented using the `Geant` toolkit.
+ The simulation step is managed by the `Gauss` application, while the digitization of the simulated events is performed by `Boole`.
+ 2. Data from the detector (or from `Boole`) are filtered through the *trigger*, which consists of L0, implemented in hardware, and HLT, implemented in software. The LHCb application responsible for the HLT is `Moore`, which will be discussed in further detail in the [trigger lesson](18-hlt-intro.html).
```
Comment (betatim) [inline 08-stripping.md]: Remove the link, create an easy issue to put it back. Two birds with one stone, we get an easy issue for people to fix and we can make Travis happy.

**Comment by betatim** [comment]
You need to link this from `index.md`

File: 08-stripping.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+To get information on the stripping, the best resource is the [stripping Twiki](https://twiki.cern.ch/twiki/bin/view/LHCb/LHCbStripping).
+In it we can find:
+
+  - The status of the current strippings, eg, for [Stripping `S21r1`](http://lhcbproject.web.cern.ch/lhcbproject/Reprocessing/stats-re-stripping-fall14.html).
+  - The configuration of all strippings, eg, for [Stripping `S21`](https://twiki.cern.ch/twiki/bin/view/LHCb/Stripping21Configuration)
+
+Additionally, the information on all strippings can be found in the [stripping project website](http://lhcb-release-area.web.cern.ch/LHCb-release-area/DOC/stripping/), where you can see all the algorithms run and cuts applied in each line.
+For example, if we wanted to understand the `StrippingD2hhCompleteEventPromptDst2D2RSLine` line, which we will use from now on, we would go [here](http://lhcb-release-area.web.cern.ch/LHCb-release-area/DOC/stripping/config/stripping21/charmcompleteevent/strippingd2hhcompleteeventpromptdst2d2rsline.html).
+
+> # Understanding a stripping line {.challenge}
```
Comment (betatim) [inline 08-stripping.md]: `#` -> `##` to make travis happy

**Pull request merged**